### PR TITLE
#30343: stop debconf from interrupting upgrades

### PR DESCRIPTION
**Pull request by magenta-af**
We don't have a PR policy for this repository yet, so I'm going to pretend that we do: pull requests and reviews are a Good Idea and we should use them for things in this repository as well.

### CODE REVIEW

**Comment by magenta-af** [comment]
I've just tested the new version of this script (as part of my tests of magenta-aps/bibos_admin#16) and it looks like it works.

File: admin_scripts/update_all.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
```
Comment (dals83) [inline admin_scripts/update_all.sh]: This script is missing our template header.

Comment (magenta-af) [reply admin_scripts/update_all.sh]: Addressed in commit 4e65effe16a7fab99deb91c8943e446ad1a5007f.

File: admin_scripts/update_all.sh [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+# END_OF_HEADER
+#================================================================
+
+set -e
+
+# Stop Debconf from doing anything
+export DEBIAN_FRONTEND=noninteractive
+
+apt-get update > /dev/null
+apt-get -y upgrade
```
Comment (dals83) [inline admin_scripts/update_all.sh]: You can add "apt-get -yf install" to fix broken dependencies.

**Pull request merged**